doc/base/classes.xml

```diff
@@ -298,7 +443,15 @@
 			<argument index="0" name="instance_id" type="int">
 			</argument>
 			<description>
-				Get an object by its ID.
+				Returns the Object that corresponds to 'instance_id'. All Objects have a unique instance ID.
+				[codeblock]
+				var foo = "bar"
+				func _ready():
+					var id = get_instance_id()
```

Comment:
Indentation inside code block should be spaces.
```
<TAB>[codeblock]
<TAB>func f():
<TAB><4SPACES>pass
<TAB>[/codeblock]
```

Reply:
ok ill fix